core/ledger/kvledger/pvtrwstorage/transient_store_helper.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright IBM Corp. 2017 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pvtrwstorage
+
+import (
+	"bytes"
+
+	"github.com/hyperledger/fabric/common/ledger/util"
+)
+
+type count uint8
+
+const (
+	NUM_ATTRIBUTES_IN_PRWSET_KEY      count = 4 //namespace~txId~endorserId~endorsementBlkHt
+	NUM_ATTRIBUTES_IN_PURGE_INDEX_KEY count = 4 //namespace~endorsementBlkHt~txId~endorserId
+)
+
+var compositeKeySep = []byte{0x00}
+
+// CreateCompositeKeyForPRWSet creates a key for storing private read-write set
+// in the transient store. The structure of the key is namespace~transactionId~endorserId.
+func CreateCompositeKeyForPRWSet(ns string, txId string, endorserId string, endorsementBlkHt uint64) []byte {
```

Comment:
Do we need these functions public? Better to hide the implementation detail within package